src/components/screens/management-screen.tsx

```diff
@@ -574,21 +581,46 @@ const ManagementScreen = () => {
     );
   }
 
+  /**
+   * Handles view change
+   * @param event 
+   * @param newTabIndex 
+   */
+  const handleTabChange = (event: ChangeEvent<{}>, newTabIndex: string) => {
```

Comment:
It looks like the event is not used here, do we still have to have it as a parameter. If so, you could name it as _event to indicate it is not used.